libraries/standard/http/src/http_client.c

```diff
@@ -3,59 +3,75 @@
 
 #include "http_client.h"
 #include "private/http_client_internal.h"
-#include "private/http_client_parse.h"
-#include "http_parser/http_parser.h"
 
 /*-----------------------------------------------------------*/
 
 /**
- * @brief An aggregator that represents the user-provided parameters to the
- * #HTTPClient_ReadHeader API function. This will be used as context parameter
- * for the parsing callbacks used by the API function.
+ * @brief Send HTTP bytes over the transport send interface.
+ *
+ * @param[in] pTransport Transport interface.
+ * @param[in] pBuffer HTTP request data to send.
```

Comment:
Parameters can be named `pData` and `dataLength`. Using buffer gives the impression as if it is an output parameter to be filled in by the function.